graphql/integration_atomic_test.go

```diff
@@ -40,6 +40,16 @@ type atomicGraphQLState struct {
 }
 
 func TestAtomicGQLQueries(t *testing.T) {
+	// ctx, cancel := context.WithCancel(context.Background())
+	// defer cancel()
+	ctx := context.Background()
+
+	env := testutil.NewEnvironment(ctx, t)
```

Comment:
Instead of doing this, import the testutil package with "_" as its alias. Doing so will run the code in its init() function, which includes this as well as some additional setup for tests (sorry I had forgotten that package existed)

Reply:
lol

Reply:
that did not work

Reply:
Could you explain what "did not work" means?

Reply:
https://evergreen.mongodb.com/task_log_raw/evergreen_ubuntu1604_test_graphql_patch_8a5071b6682cdc94d3b9613438a19b0ce1d343f2_5e726ecb0ae6067deefd59df_20_03_18_18_56_19/0?type=T#L456

Reply:
I suspect you still need to do the things that https://github.com/evergreen-ci/evergreen/blob/2603b24f129861c557078182ccd3536882b48519/testutil/config.go#L38-L51 does _not_ do. I believe that includes calling `testutil.ConfigureIntegrationTest(t, settings, "TestAtomicGQLQueries")`, and probably creating `settings` by calling `testutil.TestConfig().`

Reply:
hm. still not working from same error. https://evergreen.mongodb.com/task/evergreen_ubuntu1604_test_graphql_patch_8a5071b6682cdc94d3b9613438a19b0ce1d343f2_5e727e7932f41772e1594452_20_03_18_20_04_05

the previous way seemed to work fine